driver_test.go

```diff
@@ -2948,6 +2948,9 @@ func TestRowsColumnTypes(t *testing.T) {
 
 		values := make([]interface{}, len(tt))
 		for i := range values {
+			if types[i] == nil {
+				continue
+			}
```

Comment:
This doesn't make sense at all. Remove it.

Reply:
agree